composer.json

```
@@ -32,38 +32,41 @@
         }
     ],
     "extra": {
-        "zf": {
+        "laminas": {
             "config-provider": "DoctrineORMModule\\ConfigProvider",
             "module": "DoctrineORMModule"
         }
     },
     "require":           {
-        "php":                               "^7.1",
+        "php":                               "^7.2",
         "doctrine/doctrine-module":          "^2.1.3",
```

```suggestion
        "doctrine/doctrine-module":          "^3.0.1",
```
doctrine/doctrine-module:2.1.10 requires zendframework/* packages

---

I guess this change would still be necessary for merging. Otherwise there would be conflicts upon installing this module. Or is there a trick I am missing?